baseapp-auth/baseapp_auth/rest_framework/register/serializers.py

```diff
@@ -26,6 +28,13 @@ def validate_referral_code(self, referral_code):
                 raise serializers.ValidationError(_("Invalid referral code."))
         return referral_code
 
+    def validate_name(self, name):
```

Comment:
For me this is a bad pattern, some people do have punctuation on their names. Can we remove it?